README.md

```diff
@@ -1,10 +1,17 @@
 Infusion Documentation
 ======================
 
+## What is infusion and how do I get it? ##
+
+See the infusion website at:
+
+<http://fluidproject.org/infusion.html>
```

Comment:
Also, @kaspermarkus, it would be great to add this "what is it and where do I get it" info to the landing page that shows up in the generated docs, here: http://docs.fluidproject.org/infusion/ This is in the source file src/documents/index.md